src/main/java/life/qbic/omero/BasicOMEROClient.java

```diff
@@ -65,38 +65,40 @@
 /////////////////////////////////////////////////////
 
 /**
- * A client to handle operations on the OMERO server
+ * A client object to handle operations on the OMERO server
+ *
  *
- * This client can connect to the omero server.
  *
  * @since: 1.0.0
  */
 public class BasicOMEROClient {
 
-  //////////////////
   private final String hostname;
   private final int port;
   private final String username;
   private final String password;
   private final Gateway gateway;
   private final int serverId;
-  //////////////////
+
+  private String userGroupName;
+
   private String sessionId;
   private String sessionUuid;
   private SecurityContext securityContext;
 
-
   private HashMap<Long, String> projectMap;
   private HashMap<Long, Set<DatasetData>> datasetMap;
 
-  public BasicOMEROClient(String username, String password, String hostname, int port) {
+  public BasicOMEROClient(String username, String password, String hostname, int port, String userGroupName) {
```

Comment:
Please provide Javadoc for the public constructor (I think you can re-use stuff from the connect method)